airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_task_instances.py

```diff
@@ -4006,3 +4006,140 @@ def test_should_return_empty_list_for_updating_same_task_instance_state(
         )
         assert response.status_code == 200
         assert response.json() == {"task_instances": [], "total_entries": 0}
+
+
+class TestDeleteTaskInstance(TestTaskInstanceEndpoint):
+    ENDPOINT_URL = "/dags/example_python_operator/dagRuns/TEST_DAG_RUN_ID/taskInstances/print_the_context"
+    DAG_ID = "example_python_operator"
+    TASK_ID = "print_the_context"
+    RUN_ID = "TEST_DAG_RUN_ID"
```

Comment:
Move `ENDPOINT_URL` to the end of variable definition, and use f-string to format the URL with other DAG_ID, TASK_ID, and RUN_ID variables.